components/ws-daemon/pkg/content/service.go

```diff
@@ -165,6 +165,10 @@ func (s *WorkspaceService) InitWorkspace(ctx context.Context, req *api.InitWorks
 		if err != nil {
 			return nil, status.Errorf(codes.InvalidArgument, "invalid content manifest: %s", err.Error())
 		}
+		if req.PersistentVolumeClaim {
+			// pavel: setting wsloc as otherwise mkdir fails later on.
```

Comment:
nit

```suggestion
			// TODO(pavel): setting wsloc as otherwise mkdir fails later on.
```

see https://www.notion.so/gitpod/How-we-develop-e8ce7e562478458a9223eb9b797415b2#86e669f8e7df4f1289c9bede0855b5f9